### DISCUSSION TITLE
How to override extentions installed by admin, when I disable everything the UI shows disabled but it doesn't unblock anything.

### DISCUSSION THREAD

**Discussion by Billybob5489** [3kh0/ext-remover#602] [Help JOIN TITANIUM NETWORK TO SHARE PLEASE] [answered]
Maybe gaining developer mode inside the extensions. My school is using lightspeed filter agent version 3.2.0.1680110229.
Anything to disable that would help.

**Comment by cooltoo37** [comment, marked as answer]
first I need some info what version your chrome book do you have bookmarklets enabled and gaining dev mode would require you to use sh1mmer which would get you in trouble if school finds out so not recommended I would say provide more info then we can talk about disabling exts also have you tried all of the other ways on the main screen https://github.com/3kh0/ext-remover

**Reply by KJIsBetter** [reply]
the same thing happens to me

**Reply by FearedFusionX** [reply]
If you are running v111, the exploit doesn't work.

I tried it

**Reply by FearedFusionX** [reply]
If you are running v111, the exploit doesn't work.

I tried it
